doc/developer-guide/upgrading.rst

```diff
@@ -370,11 +370,11 @@ The following include files have been renamed:
 HTTPS support
 ^^^^^^^^^^^^^
 
-HTTPS support was moved from the core into a new module, :ref:`mod_ssl`.
+HTTPS support was moved from the core into a new module, *mod_ssl*.
```

Comment:
And mod_ssl has now been removed again.  Maybe we should delete the older upgrade notes, this gets a bit confusing.